shared/packages.sh

```diff
@@ -11,13 +11,13 @@ LIBPNG_URL="https://download.sourceforge.net/libpng/$lib-$ver.tar.xz"
 LIBPNG_DIR="$lib-$ver"
 
 lib=freetype
-ver=2.10.4
+ver=2.11.0
 FREETYPE_URL="https://download.savannah.gnu.org/releases/$lib/$lib-$ver.tar.xz"
 FREETYPE_DIR="$lib-$ver"
 FREETYPE_ARGS="--without-bzip2 --without-png --without-zlib"
 
 lib=harfbuzz
-ver=2.8.2
+ver=2.9.0
```

Comment:
We might want to migrate to meson or cmake for harfbuzz.

Reply:
the same kinda applies to pixman. They are also pushing meson, so we cannot ignore this for long anymore ^^'